app/models/commConns.js

```diff
@@ -105,6 +105,7 @@ class CommConns {
             }).catch(reject);
           }).catch(reject);
         }
+        return null
```

Comment:
Clearly, this is something I will do from now on, but curious why it is desired.